core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -372,12 +371,15 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
                 }
                 serviceResponse = HttpResponse.ofFailure(cause);
             }
+            final ServiceErrorHandler serviceErrorHandler =
+                    serviceCfg.errorHandler()
+                              .orElse(config.errorHandler().asServiceErrorHandler());
```

Comment:
Could we do this in `ServiceConfigBuilder.build()` so we don't have to call `orElse()` on every request?